conf/modules.config

```
@@ -136,6 +136,25 @@ process {
         ]
     }
 
+    withName: BBMAP_BBNORM {
+        ext.args = [
+            params.bbnorm_target ? "target=${params.bbnorm_target}" : '',
+            params.bbnorm_min    ? "min=${params.bbnorm_min}"       : '',
+        ].join(' ').trim()
+        publishDir = [
+            [
+                path : { "${params.outdir}/bbmap/bbnorm/logs" },
+                mode : 'copy',
```

I think that would be better:
```suggestion
                mode : params.publish_dir_mode,
```
also below in the second statement